provision/openshift/Taskfile.yaml

```diff
@@ -91,6 +91,61 @@ tasks:
       - .task/var-KC_STORAGE
       - .task/var-KC_DATABASE
 
+  grafana-sa:
+    # this setup was inspired by https://zhimin-wen.medium.com/custom-grafana-dashboard-for-user-workload-in-openshift-6dc2d4cad274
+    deps:
+      - common:split
+      - common:env
+      - openshift-env
+    cmds:
+      - kubectl create namespace monitoring || true
+      # this creates the service account outside the Helm chart to be able to create the token which is later an input of the Helm change
+      - oc create sa grafana -n monitoring || exit 0
+      - oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana -n monitoring
+      - bash -c 'test -e .task/grafana-sa-token || (oc create token grafana -n monitoring > .task/grafana-sa-token)'
+      - kubectl get project/monitoring -o jsonpath='{.metadata.annotations.openshift\.io/sa\.scc\.uid-range}' |  cut -f1 -d"/" > .task/monitoring-uids
+    sources:
+      - .task/subtask-{{.TASK}}.yaml
+    status:
+      - test -f .task/grafana-sa-token
+    generates:
+      - .task/grafana-sa-token
+      - .task/monitoring-uids
+
+  grafana:
+    deps:
+      - common:split
+      - common:env
+      - openshift-env
+      - grafana-sa
+    cmds:
+      - helm repo add grafana https://grafana.github.io/helm-charts
+      - >
+        helm -n monitoring upgrade --install grafana grafana/grafana --version 6.55.0 -f grafana.yaml
+        --set ingress.hosts[0]=grafana.{{.KC_HOSTNAME_SUFFIX}}
```

Comment:
Should we add this url to `isup.sh`?

Reply:
As installing the monitoring is optional for now, I chose to not add it. Reconsidering it, I could have it part of the list when it has been installed. Depending on the other findings, this could be another ticket once this has been merged.